Snippets/Snippets_6/Handlers/MyHandler.cs

```diff
@@ -15,4 +15,17 @@ public class MyAsyncHandler : IHandleMessages<MyMessage>
     }
 
     #endregion
+
+    #region EmptyHandler
+
+    public class MyMessageHandler : IHandleMessages<MyMessage>
+    {
+        public Task Handle(MyMessage message, IMessageHandlerContext context)
+        {
+            // do something in the client process
+            return Task.FromResult(0);
```

Comment:
@timbussmann no, the samples and snippets still use 4.5.2. Not sure about updating, @SimonCropp?